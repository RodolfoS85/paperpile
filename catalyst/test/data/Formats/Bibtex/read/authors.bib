@ARTICLE{au1,
  title    = {title1},
  author   = {Brennan, PJ and {on behalf of the IDEAL Consortium}},
  EDITOR   = "Tim O'Reilly",
  journal  = {Clin Vaccine Immunol},
  year	   = {2010}
}

