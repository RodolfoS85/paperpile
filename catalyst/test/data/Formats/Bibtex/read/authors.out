---
authors: 'Brennan, PJ and {on behalf of the IDEAL Consortium}'
editors: O'Reilly, Tim
